make_plugin.sh

```diff
@@ -6,6 +6,7 @@ rustc -o obj/$target/liblrs_core_plugin.so src/core_plugin/lib.rs
 
 for t in $(./targets.sh all); do
     if [[ ! -e obj/$t/liblrs_core_plugin.so ]]; then
-        ln -s ../$target/liblrs_core_plugin.so obj/$t/liblrs_core_plugin.so
+	[ -d obj/$t ] || mkdir obj/$t
+        ln -s obj/$target/liblrs_core_plugin.so obj/$t/liblrs_core_plugin.so
```

Comment:
So the link you're creating here points to

    obj/$t/obj/$target/liblrs_core_plugin.so